spec/chromium-spec.js

```diff
@@ -223,6 +223,26 @@ describe('chromium feature', () => {
       b = window.open(`file://${fixtures}/pages/window-open-size.html`, '', 'show=no')
     })
 
+    for (const show in [true, false]) {
```

Comment:
```js
for (const show of [true, false])
```

`of` iterates over array entries (kinda), `in` over an object properties.

Reply:
Lol, I love how this works regardless though, because `0` is falsey and `1` is truthy 😆

Reply:
shia_lebeouf_magic.gif